wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
WP docs say it can be a docblock: http://codex.wordpress.org/Writing_a_Plugin#Standard_Plugin_Information